app/components/leagueStandingTable/leagueStandingTable.tsx

```diff
@@ -0,0 +1,37 @@
+import { Table } from "../baseComponents";
+import { getWikipediaContestantDataFetcher } from "@/app/utils/wikiFetch";
+import generateListOfContestantRoundLists from '@/app/generators/contestantRoundListGenerator'
+import { getCompetingEntityList } from "@/app/utils/wikiQuery"
+
+import styles from "./leagueStandingTable.module.scss";
+
+export default async function LeagueStandingTable({ contestantsScores }:{contestantsScores: any}){
+    type rowState = any[];
+    const tableColNames: rowState = ["Rank", "Name", "Score"];
+    const tableRows: rowState = [];
+    const tableData = {
+        colsNames: tableColNames,
+        rows: tableRows
+    }
+
+    contestantsScores.forEach((contestantData:any) =>{
+        const { content: { props }} = contestantData;
+        const contestantRoundScores = props.contestantRoundScores;
+        const contestantRoundTotalScore = contestantRoundScores.at(-1).contestantRoundData[0].totalScore;
```

Comment:
This blind indexing, (the `-1` part at least) seems a bit error prone to me. We have a round number, why not use it? Also I think if we start leveraging the `LeagueStanding` and or one of the `generateContestantRoundScores` as mentioned above we won't need the second level of indexing (the `[0]` part).